__tests__/app.test.js

```diff
@@ -56,3 +60,37 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe('GET /api/articles/:article_id', () => {
+  test('should call this endopoint and sees if it is an object or not and if the article id matches the URL"', () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then((res)=>{
+      const article_id = res.body.article_id;
+      const author = res.body.author;
+      const title = res.body.title;
+      const topic = res.body.topic;
+      const body = res.body.body;
+      const created_at = res.body.created_at;
+      const votes = res.body.votes;
+
+      expect(typeof(res.body)).toBe("object")
+      expect(article_id).toBe(1)
+      expect(author).toBe("butter_bridge")
+      expect(title).toBe("Living in the shadow of a great man")
+      expect(topic).toBe("mitch")
+      expect(body).toBe("I find this existence challenging")
+      //expect(Date.parse(created_at)).toBe(1594329060000);
+      expect(votes).toBe(100)
+    })
+  });
+
+  test("should return the response(an object) that has the stated properties ", () => {
+    return request(app)
+      .get("/api/articles/1000")
+      .expect(404);
```

Comment:
We should also be sending an error message back here to go with the 404 status. Also you should add another test for a bad request for example `/api/articles/banana`